packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.styles.ts

```diff
@@ -214,20 +219,27 @@ export const getStyles: IStyleFunction<IDropdownStyleProps, IDropdownStyles> = p
               height: '100%',
               // see https://github.com/OfficeDev/office-ui-fabric-react/pull/9182 for semantic color disc
               border: !disabled ? `2px solid ${palette.themePrimary}` : 'none',
-              borderRadius: '2px'
-            },
-            highContrastItemAndTitleStateMixin
+              borderRadius: '2px',
+
+              selectors: {
+                [HighContrastSelector]: {
+                  borderColor: 'Highlight',
+                  color: 'Highlight'
+                }
+              }
+            }
           ],
           ['&:active .' + globalClassnames.title]: [
             !disabled && rootHoverFocusActiveSelectorNeutralDarkMixin,
             { borderColor: palette.themePrimary },
             highContrastBorderState
           ],
+          ['&:focus .' + globalClassnames.title]: [{ selectors: { [HighContrastSelector]: { color: 'Highlight' } } }],
```

Comment:
this broken line 208. pls move adding the high contrast style there

Reply:
What do you mean by "adding the high contrast style here", the `HighContrastSelector` is in there.

Reply:
i meant by you need to merge this line with line 208, otherwise it will overwrite it completely

Reply:
Oh yeah, didn't see that, I'll move this up there. Thanks!